src/channels/CredentialChannels/backgroundCredentialChannel.ts

```diff
@@ -1,5 +1,16 @@
+import type {
+  IEncryptedMessage,
+  IReject,
+} from '@kiltprotocol/kilt-extension-api/types';
```

Comment:
`@kiltprotocol/kilt-extension-api` is a funny name

Reply:
Agree!